en/drafts/originals/creating-guis-in-python-for-digital-humanities-projects.md

```diff
@@ -53,40 +40,27 @@ PDF file. The main goal is to learn the basics of creating GUIs. By the end of t
-application contains complicated features, and you do not know in what order the user might want to use those features.
-In that case, a GUI provides the freedom for the user and is actually simpler for the programmer to implement than the
-confusing and intricate logic branches and loops that a CLI script would require.
+How does one decide between programming a CLI or a GUI script/app? If the script requires little or no interaction from the user, use a CLI. If the program is relatively uncomplicated, a CLI will likely suffice. However, if you wish to share your applications or scripts with users with little programming knowledge or limited computing skills, you should use a GUI. Finally, suppose your application contains complicated features, and you do not know in what order the user might want to use those features. In that case, a GUI offers freedom for the user and is actually simpler for the programmer to implement than the confusing and intricate logic branches and loops that a CLI script would require.
 
 Our PDF Merger sample application will provide flexibility, ease of use, and efficiency that a traditional CLI application cannot achieve when
-confronting a similar research challenge. This tutorial provides a concrete example: the program’s goal is to merge various
-PDFs. A CLI could do this, but it would be challenging to merge PDFs that were not in the same file directory. With the
-GUI in this tutorial and drag-and-drop functionality, the user can select and merge specific PDFs from any file location
-on the computer, including the hard drive, cloud, USB stick, or other peripherals.
+confronted by a similar research challenge. This tutorial takes a concrete example: our program’s goal is to merge various PDFs. A CLI could do this, but it would be challenging to merge PDFs that were not in the same file directory. With the GUI created in this tutorial, the user will be able to use drag-and-drop functionality to select and merge specific PDFs from any file location on the computer, including the hard drive, cloud, USB stick, or other peripherals.
 
-After you finish the tutorial, your final design should look like Figure 1. It will be helpful to refer back to this picture as you follow the tutorial.
+After you finish the tutorial, your final design should resemble Figure 1. It may be helpful to refer back to this picture as you follow the steps.
 
-{% include figure.html filename="https://github.com/programminghistorian/ph-submissions/blob/gh-pages/images/creating-guis-in-python-for-digital-humanities-projects/CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS1.png?raw=true" caption="Figure 1. Final Design of the User Interface" %}
+{% include figure.html filename="CREATING-GUIS-IN-PYTHON-FOR-DIGITAL-HUMANITIES-PROJECTS1.png" alt="Visual description of figure image" caption="Figure 1. Final Design of the User Interface" %}
```

Comment:
Hello Christopher @DerDoktorFaust,

As part of our commitment to accessibility, we are now asking authors to provide descriptive alt-text to accompany figure images. 

You can slot the alt-text in to the template I've created here.

I've collated [some resources on our Wiki](https://github.com/programminghistorian/jekyll/wiki/Achieving-Accessibility-Alt-text-Colour-Contrast#alt-text) as guidance if you need it.

Reply:
alt="Application window with centered title text that reads PDF File Merger. Directly below is a white drag-and-drop box for files. Near the bottom of the window is a push button that reads Merge Files."

Reply:
Excellent. Thank you, @DerDoktorFaust! I've added in your alt-text throughout.